src/user/approval.js

```diff
@@ -76,7 +76,9 @@ module.exports = function (User) {
 			template: 'registration_accepted',
 			uid: uid,
 		});
-		await db.listAppend('registration:queue:approval:times', Date.now() - creation_time);
+		const total = await db.incrObjectField('registration:queue:approval:times', 'totalTime', Date.now() - creation_time);
```

Comment:
One last change, I think you can do the divide by 1000 here since we don't need to store milliseconds. Maybe you can even just store minutes in `totalTime`

Reply:
Done! Sorry for the delay - I went on to do other things and forgot about this PR till now :D

Reply:
No problem. Looks good to me, will merge after we launch 1.15.0.